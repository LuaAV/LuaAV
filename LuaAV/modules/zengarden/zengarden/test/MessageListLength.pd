#N canvas 578 89 405 361 10;
#X msg 160 137 1 2 3;
#X msg 19 137 walk the dog;
#X msg 260 137 1 x y;
#X msg 312 134 x 1 y;
#X msg 122 136 55;
#X obj 146 219 list length;
#X obj 146 249 print length;
#X obj 117 40 loadbang;
#X obj 196 40 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X obj 124 74 t b b b b b b;
#X obj 214 138 b;
#X connect 0 0 5 0;
#X connect 1 0 5 0;
#X connect 2 0 5 0;
#X connect 3 0 5 0;
#X connect 4 0 5 0;
#X connect 5 0 6 0;
#X connect 7 0 9 0;
#X connect 8 0 9 0;
#X connect 9 0 1 0;
#X connect 9 1 4 0;
#X connect 9 2 0 0;
#X connect 9 3 10 0;
#X connect 9 4 2 0;
#X connect 9 5 3 0;
#X connect 10 0 5 0;
