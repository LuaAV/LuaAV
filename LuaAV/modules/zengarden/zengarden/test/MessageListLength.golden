[@ 0.000ms] length: 3
[@ 0.000ms] length: 3
[@ 0.000ms] length: 0
[@ 0.000ms] length: 3
[@ 0.000ms] length: 1
[@ 0.000ms] length: 3